common/lib/xmodule/xmodule/public/js/vertical_student_view.js

```diff
@@ -0,0 +1,10 @@
+/* JavaScript for Vertical Student View. */
+window.VerticalStudentView = function (runtime, element) {
+
+    RequireJS.require(['js/bookmarks/views/bookmark_button'], function (BookmarkButton) {
+            var $element = $(element);
+
+            return new BookmarkButton({el: $element, bookmarked: $element.parent('#seq_content').data('bookmarked')});
```

Comment:
`el` of the `BookmarkButton` should be the element of bookmark button not the element of the XBlock. You should pass `element` as blockElement.